src/v2/Artsy/Router/buildClientApp.tsx

```diff
@@ -74,7 +74,13 @@ export function buildClientApp(config: RouterConfig): Promise<Resolve> {
           excludePaths: [
             // Due to special needs, this page has its own page-view tracking implementation.
             // @see https://github.com/artsy/force/blob/2c0db041fa6cb50e9f747ea95860ad5c38290653/src/v2/Apps/Artwork/ArtworkApp.tsx#L117-L121
-            "/artwork(.*)" 
+            "/artwork(.*)",
+          ],
+          abTestRouteMap: [
```

Comment:
cc @damassi this felt like a worthy addition (in the hopes of doing more AB tests?). It doesn't look like we've run _any_ AB tests since the app shell one...

This argument is where you can list all your running AB tests, and which routes the 'experiment viewed' event should fire.

Reply:
Yeah i dig it! If you wouldn't mind adding a test though.